nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -391,6 +391,11 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - The `erlang-ls` package no longer ships the `els_dap` binary as of v0.51.0.
 
+- The `conda` package has been updated to 24.1.2 and is now built from source. This package no longer relies on entering a shell to install miniconda. Running `conda init` is still required.
+  By default, conda environments will be installed in `~/.conda/envs` and conda packages will be cached in `~/.conda/pkgs`
+  This can be overriden at build time by modifying the `defaultEnvPath` and `defaultPkgPath` attributes respectively.
+  This can be overriden at run time by setting the `CONDA_ENVS_PATH` and `CONDA_PKGS_DIRS` environment variables.
```

Comment:
```suggestion
  This can be overriden at build time by modifying the `defaultEnvPath` and `defaultPkgPath` attributes respectively or at run time by setting the `CONDA_ENVS_PATH` and `CONDA_PKGS_DIRS` environment variables.
```